actors/badge/src/main/java/org/sunbird/badge/util/BadgingUtil.java

```diff
@@ -165,14 +167,23 @@ public static String createBadgerUrl(Map<String, Object> map, String uri, int pl
    * @return String
    */
   public static String createAssertionReqData(Map<String, Object> map) {
-    JsonObject json = new JsonObject();
-    json.addProperty(
-        BadgingJsonKey.RECIPIENT_IDENTIFIER, (String) map.get(BadgingJsonKey.RECIPIENT_EMAIL));
+
+    ObjectNode node = mapper.createObjectNode();
+    ((ObjectNode) node)
+        .put(BadgingJsonKey.RECIPIENT_IDENTIFIER, (String) map.get(BadgingJsonKey.RECIPIENT_EMAIL));
+
     if (!StringUtils.isBlank((String) map.get(BadgingJsonKey.EVIDENCE))) {
-      json.addProperty(BadgingJsonKey.EVIDENCE, (String) map.get(BadgingJsonKey.EVIDENCE));
+      ((ObjectNode) node).put(BadgingJsonKey.EVIDENCE, (String) map.get(BadgingJsonKey.EVIDENCE));
+    }
+    ((ObjectNode) node).put(BadgingJsonKey.CREATE_NOTIFICATION, false);
+
+    try {
+      return mapper.writeValueAsString(node);
+    } catch (JsonProcessingException e) {
+      ProjectLogger.log(
```

Comment:
this should not be info level logiging - append error data, as well as exception object in log function